### PR TITLE
Fix RTD workflow

### PR DESCRIPTION
**Pull request by charlesbluca**
Forgot you need to set a Git username and email in order to make the commit and push it; this should fix build failures.

### CODE REVIEW

**Review by pentschev — APPROVED** [review]
LGTM, thanks @charlesbluca .

We'll still need an approval from @rapidsai/ops-codeowners though.

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**
